src/CaptureClient/CaptureEventProcessorTest.cpp

```diff
@@ -468,25 +468,27 @@ TEST(CaptureEventProcessor, CanHandleInternedTracepointEvents) {
   tracepoint->set_cpu(2);
   tracepoint->set_tracepoint_info_key(interned_tracepoint->key());
 
-  uint64_t actual_key;
-  TracepointInfo actual_tracepoint_info;
+  uint64_t actual_key{};
+  std::optional<orbit_client_data::TracepointInfo> actual_tracepoint_info;
   EXPECT_CALL(listener, OnUniqueTracepointInfo)
       .Times(1)
       .WillOnce(DoAll(SaveArg<0>(&actual_key), SaveArg<1>(&actual_tracepoint_info)));
-  TracepointEventInfo actual_tracepoint_event;
+  std::optional<TracepointEventInfo> actual_tracepoint_event;
   EXPECT_CALL(listener, OnTracepointEvent).Times(1).WillOnce(SaveArg<0>(&actual_tracepoint_event));
 
   event_processor->ProcessEvent(interned_tracepoint_event);
   event_processor->ProcessEvent(tracepoint_event);
 
-  EXPECT_EQ(actual_key, actual_tracepoint_event.tracepoint_info_key());
-  EXPECT_EQ(actual_tracepoint_event.tracepoint_info_key(), tracepoint->tracepoint_info_key());
-  EXPECT_EQ(actual_tracepoint_info.category(), tracepoint_intern->category());
-  EXPECT_EQ(actual_tracepoint_info.name(), tracepoint_intern->name());
-  EXPECT_EQ(actual_tracepoint_event.pid(), tracepoint->pid());
-  EXPECT_EQ(actual_tracepoint_event.tid(), tracepoint->tid());
-  EXPECT_EQ(actual_tracepoint_event.time(), tracepoint->timestamp_ns());
-  EXPECT_EQ(actual_tracepoint_event.cpu(), tracepoint->cpu());
+  ASSERT_TRUE(actual_tracepoint_event.has_value());
+  EXPECT_EQ(actual_key, actual_tracepoint_event->tracepoint_id());
+  EXPECT_EQ(actual_tracepoint_event->tracepoint_id(), tracepoint->tracepoint_info_key());
+  ASSERT_TRUE(actual_tracepoint_info.has_value());
+  EXPECT_EQ(actual_tracepoint_info->category(), tracepoint_intern->category());
+  EXPECT_EQ(actual_tracepoint_info->name(), tracepoint_intern->name());
+  EXPECT_EQ(actual_tracepoint_event->pid(), tracepoint->pid());
+  EXPECT_EQ(actual_tracepoint_event->tid(), tracepoint->tid());
+  EXPECT_EQ(actual_tracepoint_event->timestamp_ns(), tracepoint->timestamp_ns());
+  EXPECT_EQ(actual_tracepoint_event->cpu(), tracepoint->cpu());
```

Comment:
I would suggest testing all the fields of `actual_tracepoint_info` first, adding an empty line, and then testing all the fields of `actual_tracepoint_event`.

Reply:
Done.